include/grpcpp/server_impl.h

```diff
@@ -199,6 +199,14 @@ class Server : public grpc::ServerInterface, private grpc::GrpcLibraryCodegen {
 
   grpc_server* server() override { return server_; }
 
+ protected:
+  void SetHealthCheckServiceInterface(
+      grpc::HealthCheckServiceInterface* service) {
```

Comment:
Any reason not to make this argument a `unique_ptr<>`?  That way the compiler can enforce ownership, and the ownership semantics of the API are clear to readers.

Reply:
Done. PTAL.